ci/scripts/e2e-source-test.sh

```diff
@@ -49,8 +49,9 @@ apt update
 apt install -y kafkacat
 cargo make clean-data
 cargo make ci-start ci-kafka
-./scripts/source/prepare_ci_kafka.sh
+./scripts/prepare_ci_kafka.sh source
 sqllogictest -p 4566 -d dev  './e2e_test/source/**/*.slt'
 
 echo "--- Run CH-benCHmark"
-./risedev slt -p 4566 -d dev ./e2e_test/ch-benchmark/ch_benchmark.slt
+./scripts/prepare_ci_kafka.sh tpcc
+sqllogictest -p 4566 -d dev  './e2e_test/ch-benchmark/ch_benchmark.slt'
```

Comment:
and I suggest move ch_benchmark to another test script
eg. ci/scripts/e2e-ch-bench-test.sh

Reply:
Yeah, they can run together on the Kafka but I figured that it would be a bit tidier to have separate folders for different tests what is your opinion? For that same reason it's indeed better to have specific script for ch-benchmark like you suggest 👍

Reply:
I am working on the separate ch-benchmark workflow here btw #5669!

Reply:
I think you have too many ongoing prs, you can look for some help. 🤣

Reply:
Due to time zone difference I sometimes need wait a couple of hours for feedback so in the meantime I can work on different PRs 😛